cli.js

```diff
@@ -3,13 +3,25 @@
 var meow = require('meow');
 var supportsColor = require('supports-color');
 
-meow({
+var cli = meow({
 	help: [
+		'Usage',
+		'  $ supports-color [--<256|16m>]',
+		'',
+		'Options',
+		'  --256     Check for 256 color support',
+		'  --16m     Check for 16 million (truecolor) support',
```

Comment:
There isn't to my knowledge. I always figured to keep it inside of 80 characters, but I see that broken all the time.